werkzeug/http.py

```diff
@@ -374,7 +374,7 @@ def parse_options_header(value, multiple=False):
             return tuple(result)
         value = rest
 
-    return tuple(result)
+    return tuple(result) if result else ('', {})
```

Comment:
Thanks. It also has to protect against values such as e.g. `` , `` so not sure if fully possible to move up in the beginning of the function but I'll give it a try.